app.py

```diff
@@ -314,7 +230,6 @@ def index():
 def handle_postback():
     today_date = datetime.now().strftime("%m/%d/%Y %H:%M")
```

Comment:
just want to flag that this app's host machine must be running in the same time zone as the knack app in order for this timestamp to be localized correctly. this is covered by the Dockerfile setting the timezone of the container image to `America/Chicago`, but ideally we would use `ZoneInfo` to make this safe to run on any system.